llvm/lib/SYCLLowerIR/SYCLVirtualFunctionsAnalysis.cpp

```diff
@@ -0,0 +1,117 @@
+//===--- SYCLVirtualFunctionsAnalysis.cpp ---------------------------------===//
```

Comment:
nit: '-` balance stuck out to me again, sorry

Reply:
Fixed in 57e43cf0fd010cd160fc9ebe9073b7c7c2711cc9